iOS/DuckDuckGo/Subscription/UserScripts/SubscriptionPagesUseSubscriptionFeature.swift

```diff
@@ -856,6 +911,8 @@ final class DefaultSubscriptionPagesUseSubscriptionFeatureV2: SubscriptionPagesU
             subscriptionParameters = frontEndExperiment.asParameters()
         }
 
+        data.activateAccountDuration = WidePixel.MeasuredInterval.startingNow()
```

Comment:
Activation measures how long it takes for us to confirm with the backend that the subscription is active, via `completeSubscriptionPurchase`.